src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -342,6 +342,9 @@
 helps['network dns zone create'] = """
     type: command
     short-summary: Creates or updates DNS zone properties. Does not modify DNS records within the zone.
+    parameters:
+        - name: --if-none-match
```

Comment:
if_match makes sense from the perspective of using the SDK's create_or_update method to conditionally overwrite something. However, since we hide the "updatey" aspect of create_or_update, it wouldn't make sense to expose if_match in this case.